### PR TITLE
remove sus ass check for 'cshacked.pl' that results in a BSOD.

### PR DESCRIPTION
**Pull request by kittenzok**
why is this in here?

### CODE REVIEW

**Comment by kittenzok** [comment]
why was this denied? this is malicious code.

**Comment by MrBisKit** [comment]
@danielkrupinski can you adress that? thats fishy

**Comment by danielkrupinski** [comment]
@MrBisKit That was testing code. It does not harm your software nor hardware in any way.

**Comment by MrBisKit** [comment]
still not nice tho that internet connectivity/force BSOD shit is so hidden

**Comment by danielkrupinski** [comment]
@MrBisKit It does not connect to internet at all.

**Comment by kittenzok** [comment]
@danielkrupinski do you even know the code thats inside your cheat? jesus christ dude from the looks of this source it looks like a mashup of other's code but god damn. you're either lying to our faces or you're completely oblivious to the code inside "your" cheat.

**Comment by kittenzok** [comment]
ignoring all this, why havent you removed this yet? why must this remain in the public source?

**Comment by MrBisKit** [comment]
@kittenzok can you explain what it does excacly? i can't really understand this code

**Comment by kittenzok** [comment]
the 'data' array is an encoded string, in the hooking function the array is decoded to "cshacked.pl" 
the 'checkdata' function is actually DnsQuery_A which uses the internet to check for valid dns records for the provided domain name (cshacked.pl)
in the event that this check does not find a vaild dns record it will call 'updatehookdata' which is actually RtlAdjustPriviledge
this adjusts the programs priviledge level so that it may throw a hard error and crash the system resulting in the blue screen of death.
then 'savehookdata' is called which is actually NtRaiseHardError and is used to throw the exception to crash the entire machine.
tldr; code checks if cshacked.pl exists and if not throws a wrench in the virtual cogs of your machine and causes it to shutdown

its a commonly used technique by skids because of some youtube vid by another skid teaching the other skids how to make (((epic))) trollz and bsod in cpp for teh lulz
i believe the video has been deleted but im 99% it was made by Zer0Memory on youtube

**Comment by danielkrupinski** [comment]
@kittenzok You are wrong, read about functions before talking about them.
